installer/PowerToysSetup/Common.wxi

```diff
@@ -17,11 +17,24 @@
 
   <?define RepoDir="$(var.ProjectDir)..\..\" ?>
   <?if $(var.Platform) = x64?>
-          <?define PlatformLK="x64" ?>
-  <?define BinDir="$(var.RepoDir)x64\$(var.Configuration)\" ?>
+        <?define PlatformLK="x64" ?>
+        <?define BinDir="$(var.RepoDir)x64\$(var.Configuration)\" ?>
   <?else?>
-          <?define PlatformLK="arm64" ?>
-  <?define BinDir="$(var.RepoDir)ARM64\$(var.Configuration)\" ?>
+        <?define PlatformLK="arm64" ?>
+        <?define BinDir="$(var.RepoDir)ARM64\$(var.Configuration)\" ?>
   <?endif?>
+
+  <?if $(var.PerUser) = "true"?>
```

Comment:
What is setting this? Installer parameter? For enterprise usage I think two things are required: Allow forcing machine install and blocking user install using a GPO. (Cc: @jaimecbernardo for the GPO.)

Reply:
> Allow forcing machine install

This is a build argument. There will be 2 separate installers. Like for VSCode.

> blocking user install using a GPO

Good point.

Reply:
@stefansjfw
The PR #24221 implements a new GPO category where you can add your installer GPO. ;-)

Reply:
It's on TODO list, It'll probably go as a follow-up PR